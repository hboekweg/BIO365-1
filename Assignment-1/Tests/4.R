source("4.R")
