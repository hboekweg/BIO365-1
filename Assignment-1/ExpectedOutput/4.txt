[1] "The number of lions in South Africa is 42. The number of tigers in South Africa is 17."
